DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/hard_drop_counters/conftest.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import pytest_asyncio
+
+from dent_os_testbed.lib.ip.ip_neighbor import IpNeighbor
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+)
+
+
+@pytest_asyncio.fixture()
+async def clear_arps(testbed):
+    """
+    Cleanup arp entries from DUT
+    """
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        pytest.skip('The testbed does not have enough dent with tgen connections')
+    dev_name = dent_devices[0].host_name
+    dut_ports = tgen_dev.links_dict[dev_name][1]
+
+    yield
+
+    out = await IpNeighbor.show(input_data=[{dev_name: [{'cmd_options': '-j -4'}]}], parse_output=True)
+    assert not out[0][dev_name]['rc'], 'Failed to get IPv4 neighbors'
+    arps = out[0][dev_name]['parsed_output']
+
+    out = await IpNeighbor.delete(input_data=[{dev_name: [
+        {'dev': arp['dev'], 'address': arp['dst']}
+        for arp in arps if arp['dev'] in dut_ports
+    ]}])
```

Comment:
You can multiply a list by a number. This way you don't need to call `IpNeighbor.delete` twice:
```suggestion
    ] * 2}])
```

```python
[1, 2, 3] * 2  # [1, 2, 3, 1, 2, 3]
```